package.js

```diff
@@ -274,15 +274,17 @@ function packageTask(pkgPath, commonDeps, commonSrc){
                 })
 
                 // Statically link the required internal common modules.
-                var taskDeps;
-                if ((taskDeps = commonDeps[task.name])) {
-                    taskDeps.forEach(function (dep) {
-                        gutil.log('Linking ' + dep.module + ' into ' + task.name);
-                        var src = path.join(commonSrc, dep.module);
-                        var dest = path.join(tgtPath, dep.dest);
-                        shell.mkdir('-p', dest);
-                        shell.cp('-R', src, dest);
-                    })
+                if(commonDeps){
```

Comment:
Hi @ericsciple I went in error when calling pkgm.PackageTask() without commonDeps param on a custom gulp build task. I think that, since the param is optinal, would be better check to avoid errors (just to keep things SOLID).